Task.js

```diff
@@ -171,25 +169,31 @@ class Task extends Unit {
         break;
 
       case ContentTypes.ASSESMENT:
-      	  page_data.isAssesment = true;
-      if(shell.test("-f", this.sourcePath())){
-        page_data.questions = require(this.sourcePath());
-        page_data.questions_str = JSON.stringify(page_data.questions);
-	page_data.isJsonVersion = true;
-      }
-      else {
-        const jsonpath = this.sourcePath();
-	const jspath = path.resolve(path.dirname(jsonpath), `${path.basename(jsonpath, "json")}js`);
-	
-	if(shell.test("-f", jspath)){
-          page_data.quiz_src = path.basename(jspath);
-	  page_data.isJsVersion = true;
-	}
-	else {
-	  console.log(`${jspath} is missing`);
-	  process.exit(-1);
-	}
-      }
+        page_data.isAssesment = true;
+        if (shell.test("-f", this.sourcePath())) {
+          page_data.questions = require(this.sourcePath());
+          if (page_data.questions.version) {
+            if (page_data.questions.version == 2)
+              page_data.jsonVersion = page_data.questions.version;
+            page_data.questions = page_data.questions.questions;
+          }
```

Comment:
My reasoning here sir, minor version support exists, but 2.0 from json comes as 2 to js. If we want to do a bump to 2.1 it wll still work so I suppose that should be fine. 
for line 178, that is fall back, just in case version is mentoined but it is a not a supported version 
For eg. 3.0 or 2.5, we'll fall back and display the questions normally.

Reply:
That would mean that the questions will start to be displayed in the old way in case someone bumps up the json version but does not update the code. I understand your intention here but the solution doesn't appear good to me.

Reply:
Sir it's kinda a bit different, i.e. if an incorrect version is mentioned, it'll fallback to old method. Maybe we can work on a better way for this through schema validation.
as such if version 2 is provided and the json is partly complete it'll still use new script for now